cmd/operator-sdk/add/controller.go

```diff
@@ -32,11 +32,17 @@ func newAddControllerCmd() *cobra.Command {
 		Use:   "controller",
 		Short: "Adds a new controller pkg",
 		Long: `operator-sdk add controller --kind=<kind> --api-version=<group/version> creates a new
-controller pkg under pkg/controller/<kind> that, by default, reconciles on a custom resource for the specified apiversion and kind.
-The controller will expect to use the custom resource type that should already be defined under pkg/apis/<group>/<version>
-via the "operator-sdk add api --kind=<kind> --api-version=<group/version>" command.
-This command must be run from the project root directory.
-If the controller pkg for that Kind already exists at pkg/controller/<kind> then the command will not overwrite and return an error.
+controller pkg under pkg/controller/<kind> that, by default, reconciles on a custom resource for the specified
+apiversion and kind. The controller will expect to use the custom resource type that should already be defined 
+under pkg/apis/<group>/<version> via the operator-sdk add api --kind=<kind> --api-version=<group/version> command. 
+
+**IMPORTANT:** This command must be run from the project root directory.
+		
+**NOTE:** If the controller pkg for that Kind already exists at pkg/controller/<kind> then the command will not 
+overwrite and return an error.
+		
+The following example will create a controller to manage, watch and reconcile as primary resource the 
+<v1alpha1.AppService> from the domain <app.example.com>.    
```

Comment:
We should move the examples to the `cobra.Command.Example` field while we're at it.